packages/sui-mono/package.json

```diff
@@ -10,11 +10,24 @@
   "dependencies": {
     "@s-ui/helpers": "1",
     "commander": "8.3.0",
-    "conventional-changelog": "3.1.25",
     "enquirer": "2.3.6",
     "git-url-parse": "13.1.0",
     "glob": "8.0.3",
-    "word-wrap": "1.2.4"
+    "word-wrap": "1.2.4",
+    "add-stream": "^1.0.0",
+    "conventional-changelog-writer": "^5.0.0",
+    "conventional-commits-parser": "^3.2.0",
+    "dateformat": "^3.0.0",
+    "get-pkg-repo": "^4.0.0",
+    "git-raw-commits": "^2.0.8",
+    "git-remote-origin-url": "^2.0.0",
+    "git-semver-tags": "^4.1.1",
+    "lodash": "^4.17.15",
+    "normalize-package-data": "^3.0.0",
+    "q": "^1.5.1",
+    "read-pkg": "^3.0.0",
+    "read-pkg-up": "^3.0.0",
+    "through2": "^4.0.0"
```

Comment:
The `conventional-changelog-core` dependencies

Reply:
Do we want to have fixed versions?